fdbserver/workloads/TenantManagementConcurrencyWorkload.actor.cpp

```diff
@@ -171,14 +171,15 @@ struct TenantManagementConcurrencyWorkload : TestWorkload {
 	ACTOR static Future<Void> createTenant(TenantManagementConcurrencyWorkload* self) {
 		state TenantName tenant = self->chooseTenantName();
 		state TenantMapEntry entry;
-		entry.tenantName = tenant;
-		entry.tenantGroup = self->chooseTenantGroup();
+		state MetaclusterTenantMapEntry mEntry;
+		entry.tenantName = mEntry.tenantName = tenant;
+		entry.tenantGroup = mEntry.tenantGroup = self->chooseTenantGroup();
```

Comment:
I'll let you decide, but another option for writing this would be to use the new `TenantMapEntry` constructor:

```
state MetaclusterTenantMapEntry entry;
...
TenantAPI::createTenant(self->dataDb.getReference(), tenant, TenantMapEntry(entry));
```

Reply:
Unfortunately this breaks the `setId` assertion of `id >= 0` so I'll be reverting this

Reply:
Where is this? Both `MetaclusterAPI::createTenant` and `TenantAPI::createTenant` should allow you to use a tenant ID of -1

Reply:
Perhaps something else is broken that needs fixed